content/Objects/Objects.facts.md

```diff
@@ -0,0 +1,37 @@
+### Facts for Objects lesson:
+
+An object is the actual representation or instance of a class.
+
+The `new` keyword is used to create an object as an instance of a class.
+
+`Instantiation` is the process of creating a new object of a class.
+
+To create an object, write the name of the object followed by an equal sign `=`; the `new` keyword and its corresponding class name. One way to write it is this way:
```

Comment:
![replaceIsWorrd_OOP_Objects](https://user-images.githubusercontent.com/38550259/60631581-b2fd2a00-9e32-11e9-86c3-6cd61a46b0b8.jpg)

Kindly check this out. Let me know what you think. Thanks.

Reply:
Sorry, but I have to retain this one. I don't know how you test it but there is no problem with my testing result and if we are trying to evaluate the statement, I think it is not right to say `One wat to write it in this way` it should be __`One way to write it is this way`__ and we are having the same statement of this already in our previous lesson.
![image](https://user-images.githubusercontent.com/25945790/60632612-2efa7080-9e39-11e9-9909-9016bac6d91e.png)